.vscode/settings.json

```diff
@@ -16,5 +16,6 @@
     "mocha.enabled": true,
     "omnisharp.autoStart": false,
     "editor.formatOnSave": false,
-    "eslint.lintTask.enable": true
+    "eslint.lintTask.enable": true,
+    "dotnet.defaultSolution": "disable"

```

Comment:
what's this about?

Reply:
Ah this is useful - prevents devkit from trying to activate on the random test assets we hgave.

Reply:
Yep, there's no "normal" C# solution in this repository, but a bunch of test projects. So without this any time you open the repository Dev Kit will prompt encouraging to open one of those, which you never need to do.

Reply:
thaks!

Reply:
This bit ultimately got taken as another PR around that time.